internal/utils/utils.go

```diff
@@ -32,3 +37,34 @@ func UnwrapInterfaceToPointer[T any](i interface{}) *T {
 func PointOf[T any](value T) *T {
 	return &value
 }
+
+func GetAccountLedgerKey(address string) (string, error) {
+	decoded, err := strkey.Decode(strkey.VersionByteAccountID, address)
+	if err != nil {
+		return "", err
+	}
+	var key xdr.Uint256
+	copy(key[:], decoded)
```

Comment:
Why are we copying `decoded` into `key`?

Reply:
Because `decoded` is a byte array and `key` is a fixed size [32]byte array. In order to convert slice to fixed size, the recommended way would be to use `copy()`: https://stackoverflow.com/questions/30285680/how-to-convert-slice-to-fixed-size-array.

We can also use something like this - `key = *(*[32]byte)(decoded)` but `copy()` would be cleaner